### PR TITLE
bluez: bump to 5.47 fix CVE-2017-1000250

### PR DESCRIPTION
**Pull request by nxhack**
bluez: bump to 5.47 fix CVE-2017-1000250

Maintainer: @psycho-nico 
Compile tested: mips_24kc_gcc-6.3.0_musl, LEDE head r4901-ab26fc6
Run tested: none

Description:
update to 5.47
Fix for CVE-2017-1000250 (part of the recently announced BlueBorne vulnerabilities).

http://www.bluez.org/release-of-bluez-5-47/

I referred to @artynet 's work
https://github.com/artynet/LEDE/tree/linino-37/package/linino/bluez

Signed-off-by: Hirokazu MORIKAWA <morikw2@gmail.com>

### CODE REVIEW

File: utils/bluez/Makefile [hunk 1]
```diff
@@ -70,7 +70,7 @@ define Package/bluez-daemon/conffiles
 /etc/bluetooth/main.conf
 /etc/bluetooth/network.conf
 /etc/bluetooth/input.conf
-/etc/bluetooth/proximity.conf
```
Comment (champtar) [inline utils/bluez/Makefile]: why are you removing proximity.conf? are you sure there is no useful information in this conf file (if you upgrade 2 times the router this file will be lost)

**Comment by artynet** [comment]
@champtar proximity.conf does exist anymore in the source tarball and I will get a packaging error if keeping it in the makefile. I decided to drop it for this very reason since the bluetooth stack seems not using it anymore.

Best, Arturo

**Comment by nxhack** [comment]
Thank you! @artynet

**Comment by artynet** [comment]
Hi @nxhack, I think we should exchange contacts for making a couple of interesting things together...do you agree ?

Please write me to arty.net2@gmail.com so we can discuss it privately (we should exchange skype IDs as well)

Best, Arturo

**Comment by nxhack** [comment]
@artynet 
It looks fun :). I will send you a mail later.

**Pull request merged**